beancount/utils/encryption.py

```diff
@@ -33,7 +33,7 @@ def is_encrypted_file(filename):
     if ext == ".gpg":
         return True
     if ext == ".asc":
-        with open(filename) as encfile:
+        with open(filename, encoding="utf-8") as encfile:
```

Comment:
That one should also be None or "locale" I think

Reply:
Asc file is pure ascii file, right?

Reply:
I think the encoding of the contents are base64 (off the top of my head)

Reply:
Then it's ok to use utf-8 I think

Reply:
I think it should be `"ascii"`.

Reply:
we should open file in ascii encoding and suppress UnicodeDecodeError here.

Reply:
An armored GPG encrypted file is a base64 encoded binary. It is guaranteed to contain only ascii characters. Do you have evidence of the contrary? Why do you want to suppress decoding errors?

Reply:
Oh. Ok. This function is probing whether the file is or is not an armored GPG encrypted file. It makes sense to do not raise an exception if the file does not have the expected encoding. It looks ok.